packages/client/src/clients/feed/feed.ts

```diff
@@ -750,6 +739,23 @@ class Feed {
     );
   }
 
+  private emitEvent(
+    type:
+      | MessageEngagementStatus
+      | "all_read"
+      | "all_seen"
+      | "all_archived"
+      | "unread"
+      | "unseen"
+      | "unarchived",
+    items: FeedItem[],
+  ) {
+    // Handle both `items.` and `items:` format for events for compatibility reasons
+    this.broadcaster.emit([`items.${type}`, `items:${type}`], { items });
```

Comment:
I assume this does the same thing emitting each one like below?
```
this.broadcaster.emit(`items.${type}`, { items });
this.broadcaster.emit(`items:${type}`, { items });
```

Reply:
I actually just checked and I was wrong so fixed this!